src/opensles/AudioOutputStreamOpenSLES.cpp

```diff
@@ -343,6 +343,11 @@ Result AudioOutputStreamOpenSLES::requestPause() {
  * Flush/clear the queue buffers
  */
 Result AudioOutputStreamOpenSLES::requestFlush() {
+    std::lock_guard<std::mutex> lock(mLock);
+    return requestFlush_l();
+}
+
+Result AudioOutputStreamOpenSLES::requestFlush_l() {
```

Comment:
Does the `_l` mean: this function should be called under a lock?

Reply:
Yes, that is a convention in Android framework.